articles/extensions/custom-social-extensions.md

```diff
@@ -92,3 +102,19 @@ If the identity provider requires Basic Authentication, you can be use the __Cus
     "Authorization" : "Basic MTIzNDU2OmFiY2RlZg=="
 }
 ```
+
+## Additional Provider Resources

```

Comment:
You're right, I removed this and added a sentence about searching for the provider's docs.